extensions/amp-ad/0.1/amp-ad-3p-impl.js

```diff
@@ -127,6 +133,21 @@ export function isPositionFixed(el, win) {
   return hasFixedAncestor;
 }
 
+/**
+ * @param {!Element} el
+ * @param {!Window} win
+ * @return {string|null} a string that contains all containers of the ad.
+ * This is called before creating iframe for this ad.
+ */
+export function getAdContainerList(el) {
```

Comment:
As discussed in #3815, we can start with simple case: only check the immediate parent. It fulfills the needs right now, and saves us some CPU cycles.

We can always add list support later when necessary.